package.json

```diff
@@ -44,14 +44,15 @@
     "postinstall": "husky install"
   },
   "dependencies": {
-    "@nestjs/common": "^9.0.0",
+    "@nestjs/common": "^10.0.2",
     "@nestjs/config": "^3.0.0",
-    "@nestjs/core": "^9.0.0",
-    "@nestjs/platform-express": "^9.0.0",
-    "@nestjs/swagger": "^6.3.0",
+    "@nestjs/core": "^10.0.2",
+    "@nestjs/platform-express": "^10.0.2",
+    "@nestjs/swagger": "^7.0.2",
     "@node-redis/json": "^1.0.2",
     "@types/lodash": "^4.14.191",
     "ajv": "^8.11.0",
+    "ajv-formats": "^2.1.1",
```

Comment:
Due to peer dependencies, we need install `ajv-formats`. The [actual upgrade of `ajv`](https://github.com/safe-global/safe-client-gateway-nest/pull/451) deals with this as well.